src/main/java/com/ibm/inventory_management/services/StockItemService.java

```diff
@@ -23,8 +23,7 @@
 
 //@Profile("!mock")
```

Comment:
Why has this been commented out? The idea was that the Profiles would determine which instance was loaded. The problem is that `@Primary` is a static and would require a code change to alter the implementation. Profiles can be applied at runtime in the configuration

Reply:
I didn't commend out the line.  What I am trying to achieve

1. As per the " Inventory Part1 Learning"    document, there is no cloudant integration. So if the developer checkout solution and try to run the solution it will not run. Because we need to uncomment the above line or change the document. 

2. If the developer walkthrough code she/he might be confused because of the cloudant integration.  Because it's not covered in Part 1 Learning journey. So either we have update document or remove cloudant integration. Please advice

Reply:
We need to decide if the solution is for the entire process or if there will be multiple versions of the solution. I'm very much inclined to have 1 solution for the entire process. It is hard enough keeping the one version up to date, much less multiple versions.

Perhaps we should update the README to describe the different components of the solution and in which module they are introduced (e.g. CloudantService is built in learning module X)

Reply:
I agree with @seansund the solution was always intended for people wanting to skip the gradual build up of the steps in the material. So it should include all the code.

Reply:
There should be atleast two solutions in my opinion, 1) readymade solution at the end of part 1 2) readymade solution at the end of part 2. The documentation is written for user to start directly from part 2 if needed, unless that is completely removed now.